test/unit/specs/components/governance/__snapshots__/PageProposal.spec.js.snap

```diff
@@ -94,78 +94,77 @@ exports[`PageProposal has the expected html structure 1`] = `
             />
              
           </div>
-        </div>
-      </div>
-       
-      <menu
-        class="tm-page-header-menu"
-      >
-        <div>
-          <div>
-            <div
-              class="tm-tool-bar"
+           
+          <div
+            class="tm-tool-bar"
+          >
+            <a
+              class="back"
+              disabled="disabled"
             >
-              <a
-                class="back"
-                disabled="disabled"
-              >
-                <i
-                  class="material-icons"
-                >
-                  arrow_back
-                </i>
-              </a>
-               
-              <router-link-stub
-                exact="exact"
-                to="/governance"
+              <i
+                class="material-icons"
               >
-                <i
-                  class="material-icons"
-                >
-                  arrow_back
-                </i>
-              </router-link-stub>
-              <a
-                class="help"
+                arrow_back
+              </i>
+            </a>
+             
+            <!---->
+             
+            <!---->
+              
+            <a
+              class="help"
+            >
+              <i
+                class="material-icons"
               >
-                <i
-                  class="material-icons"
-                >
-                  help_outline
-                </i>
-              </a>
-               
-              <router-link-stub
-                id="settings"
-                to="/preferences"
+                help_outline
+              </i>
+            </a>
+             
+            <router-link-stub
+              id="settings"
```

Comment:
it weird we're using settings and preferences instead of sticking to one or the other (other issue)

Reply:
Agree! The id should be `preferences` as we are going to that route, the only thing that leave us related to the word `settings` will be the material icon name